app/services/decisions/case_decision_tree.rb

```diff
@@ -108,7 +108,11 @@ def after_hearing_details
     # rubocop:disable Style/GuardClause
     def after_ioj
       if Passporting::MeansPassporter.new(current_crime_application).call
-        edit('/steps/submission/review')
```

Comment:
Can we for now, just ignore feature flags etc.? Do not introduce any decision in previous trees at all.
Trying to self contained this to the absolute minimum without polluting any other parts of the application.